app.js

```diff
@@ -85,7 +85,7 @@ io.on('connection', function(socket) {
 		for(var i = 0; i<num_rooms; ++i) {
 
 			//if room name is taken
-			if(rooms[i].name == data.room_name) {
+			if(rooms[i].name.val().toUpperCase() === data.room_name.val().toUpperCase()) {
```

Comment:
Any specific reason to change this? It is creating bug that modal doesn't collapses.